angular-rails-app/app/controllers/owners_controller.rb

```diff
@@ -3,10 +3,15 @@ class OwnersController < ApplicationController
    before_action :set_owner, only: [:show]
 
   def index
-    @owners = Owner.all
+    if params[:name]
+       @owners = Owner.where("name LIKE (?)", "%#{params[:name]}%").limit(6)
```

Comment:
@chrishenry:  Ok, I've added the `OR` statement in the Rails query, so that we include matches from both the `name` and `corporation_name`columns.  In addition, I am concatenating `name` + `corporation_name` to display both together in the front end dropdown box.  Works pretty well! However, per my recent message on Slack, `corporation_name` is currently type: integer.  I've written a new migration to change this to a string.  Is that the intention?  (Just waiting on confirmation of string vs integer before pushing my code.)